src/js/tech/flash.js

```diff
@@ -268,6 +268,16 @@ class Flash extends Tech {
   }
 
   /**
+   * Get the last error object
+   *
+   * @return {Object}
+   * @method error
+   */
+  error() {
+    return this.error_ || {};
```

Comment:
Technically the player event handler would only be called when an error did exist, but if we want to be save we can add a check for null. The `tech.error()` getter might be called outside of an error event.